tests/unit/PartialSubmissionControllerTest.php

```diff
@@ -53,6 +56,23 @@ public function testSavePartialSubmissionFieldCreated()
         $this->assertCount(1, $fields);
     }
 
+    public function testSavePartialSubmissionFileField()
+    {
+        $form = $this->objFromFixture(UserDefinedForm::class, 'form1');
```

Comment:
This form is already published in the `setUp()`

Reply:
D'oh :D